docker-compose.yml

```diff
@@ -74,6 +74,17 @@ services:
         constraints: [node.role == manager]
     networks:
       - webnet
+  dbmanager:
+    image: johnyburd/dbmanager:${TAG:-latest}
+    build: ./dbmanager
+    environment:
+      PGUSER: autbot
+      PGPASSWORD: autism
+      PGDATABASE: autbot
```

Comment:
unless you need these exact names, use the variables in `architus.env` instead

Reply:
The only one I need is PGPASSWORD, but I might pull it out of here to keep it dry if i use enviromental variables anyways

Reply:
please do